### ISSUE TITLE
missing thirdpartylibs.xml file.

### ISSUE THREAD

**Issue by danmarsden** [jmulet/tiny_codepro#3]
When including an external library in your plugin, you must include a thirdpartylibs.xml file that includes the name, location and license of the library.
More information on this is here:
https://docs.moodle.org/dev/Plugin_files#thirdpartylibs.xml
And here:
https://docs.moodle.org/dev/Plugin_with_third_party_libraries

One of the other advantages of using this is that Moodle's codechecker automatically ignores any files included in the location specificed in the thirdpartylibs.xml file.

Please note - this is a blocker for approval in the plugins db.

**Comment by danmarsden** [comment]
sorry - old links... right ones: 
https://moodledev.io/docs/apis/commonfiles#thirdpartylibsxml

https://moodledev.io/general/community/plugincontribution/thirdpartylibraries

**Comment by jmulet** [comment]
This should not be a problem. 
I have checked that codeMirror is distributed under a permissive MIT license that is compatible with GNU GPL.
I will add the file thirdpartylibs.xml in the root of the plugin directory. Is it ok?

**Comment by danmarsden** [comment]
yeah - it's not a problem adding 3rd party libs to your plugin, it just needs to include documentation in the form of the thirdpartylibs.xml file as detailed in the links above. I note in your develop branch you have listed codemirror twice in your new thirdpartylibs.xml file - you only really need to list it once - you don't need to separate out different parts of the 3rd party lib - just list where it is in your plugin (so the codechecker can ignore it), what license it uses, and what version of the library you have used.

when you're ready just upload the new version to the plugins db and hit the request re-approval button and I'll take another look! :-)
